pom.xml

```diff
@@ -175,6 +175,24 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-job</artifactId>
+      <version>2.39</version>
+      <scope>compile</scope>
```

Comment:
Compile scope here sounds dangerous. It will prevent Jenkins from handling the dependencies properly. workflow-job is already added above, problem is with versions

Reply:
I made new PR with fix #99